mvpa2/algorithms/group_clusterthr.py

```diff
@@ -194,6 +216,18 @@ class GroupClusterThreshold(Learner):
             probabilities. All methods supported by statsmodels' ``multitest``
             are available. In addition, ``None`` can be specified to disable
             correction.""")
+    # If default metric would be changed from 'cluster_sizes' to some fw metric
+    # make multicomp_correction default to None
+
+    qe = Parameter(
+        None,  #  constraints=  NOT SURE TODO
+        doc="""``QueryEngine`` (as used by searchlights) to determine feature's
+        immediate neighborhood.  If None provided, `ds.a.mapper` will be used to
+        reverse data into original possibly multi-dimensional shape.  If `ds`
+        does not have a mapper, 1D neighborhood along the features vector will
+        be used
+        """)
+    # TODO: relevant only for cluster-based analyses.
```

Comment:
@hanke a similar approach is used in CoSMoMVPA. But, one has to be careful that the query engine use the same space (e.g. voxels) and the same feature attributes. 
In CoSMoMVPA this is achieved by storing the .fa and relevant .a fields in the "query engine" (called neighborhood in CoSMo), so that when a neighborhood is used for clustering a dataset, it can be verified that these fields match.
See https://github.com/CoSMoMVPA/CoSMoMVPA/blob/master/mvpa/cosmo_cluster_neighborhood.m